frontend.java

```diff
@@ -0,0 +1,540 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
+public class frontend {
+	static int loggedIn = 0;
+	static int type = 0; // if type = 1 its retail, if type=2 its agent
+	static ArrayList<String> transactionFile = new ArrayList<String>();
+	public static void main(String[] args) {
+		login();
+	}
+	public static void login(){
```

Comment:
This functions does two things:
1. Enter command
2. if command is login, enter user type.

As a result, this should have been split into two methods: (1) process the type of commands requested, and (2) process the login type.